Tests/TuistGeneratorIntegrationTests/Generator/TestData/GeneratedProjectDescriptor+TestData.swift

```diff
@@ -0,0 +1,31 @@
+
+import Basic
+import Foundation
+import XcodeProj
+@testable import TuistGenerator
+
+extension ProjectDescriptor {
```

Comment:
I'd move this one to `TuistGeneratorTesting` so that the tests of the targets that depend on `TuistGenerator` can use them too.